openemrs/config/environments/production.rb

```diff
@@ -55,8 +55,8 @@
 
   config.action_mailer.perform_caching = false
 
-  # Ignore bad email addresses and do not raise email delivery errors.
-  # Set this to true and configure the email server for immediate delivery to raise delivery errors.
+  # Ignore bad email addresses and do not raise email delivery error.
+  # Set this to true and configure the email server for immediate delivery to raise delivery error.
```

Comment:
I think we can do without this change. For two reasons

1. It's English semantics
2. What happens when there are multiple errors concerning email?

Reply:
I think this was introduced by running an initializer. Not sure which but would revert.